src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -156,7 +156,7 @@ func runClientProxy(cfg *config.Config) {
 	proxiedsites.Configure(cfg.ProxiedSites)
 
 	// launching geolookup service when in client mode.
-	go geolookup.StartService()
+	go geolookup.StartService(hqfd.DirectHttpClient())
```

Comment:
We pass the fronted.Dialer from client.Configure() to the geolookup service.  This way, client doesn't directly have a dependency on the geolookup service.